module/VuFind/src/VuFind/Db/Row/Tags.php

```diff
@@ -42,7 +43,7 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Site
  */
-class Tags extends RowGateway implements \VuFind\Db\Table\DbTableAwareInterface
+class Tags extends RowGateway implements \VuFind\Db\Table\DbTableAwareInterface, TagsEntityInterface
```

Comment:
Have to admit that I've always found the use of imports vs qualifiers in VuFind a bit confusing. 😅

Reply:
I'm certainly open to making this more uniform if there's a php-cs-fixer rule we could use to enforce it. At present, it's largely driven by what takes up the least space -- I include an import if something is used many times or would cause overly long lines, but I'll use a qualifier if it's used just once in an otherwise short line of code. Not claiming this is the best way, just the habit of many years. :-)